sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -2031,6 +2058,15 @@ public AccumT extractOutput(KV<K, Integer> key, AccumT accumulator, Context c) {
                   throws CannotProvideCoderException {
                 return accumCoder;
               }
+              @Override
+              public void populateDisplayData(DisplayData.Builder builder) {
+                super.populateDisplayData(builder);
+                builder.add(DisplayData.item("fanoutFn", hotKeyFanout.getClass())
+                    .withLabel("Fanout Function"));
+                if (hotKeyFanout instanceof HasDisplayData) {
```

Comment:
I think I understand what you did here. The hot key display data now goes into the hot key fn (which is now a SimpleFunction instead of SerializableFunction).

Rather than re-implementing display data, can you use the display data registered in the `keyedFn` this is wrapping?

```java
@Override
public void populateDisplayData(DisplayData.Builder builder) {
  keyedFn.populateDisplayData(builder);
}